src/String/Extra.elm

```diff
@@ -0,0 +1,22 @@
+module String.Extra exposing (isCapitalized)
+
+{-| Some utilities.
+-}
+
+
+{-| Check if the first character of a string is upper case, unicode aware.
+
+Note that `Char.isUpper` is ASCII only, while `Char.isUpper` works outside ASCII.
+
+-}
+isCapitalized : String -> Bool
+isCapitalized string =
+    case String.uncons string of
+        Just ( char, _ ) ->
+            -- 1. The character has a lower and upper variant.
+            (Char.toUpper char /= Char.toLower char)
+                -- 2. The character is its upper variant.
+                && (Char.toUpper char == char)
```

Comment:
The fix for the core libraries PR from https://github.com/elm/core/pull/1138/files is

```suggestion
            -- 1. The character is its own upper variant.
           (char == Char.toUpper char)
                -- 2. The character is not its own lower variant.
                && (char /= Char.toLower char)

```

But I'm not sure if it really is important if `Char.toLower` is called twice here.

Reply:
What do you think @lydell? This version does seem a bit better I'd say?

Reply:
Yes, one less `Char.toUpper` call :+1: